cfgov/jinja2/v1/youth_employment_success/route-option-form.html

```diff
@@ -15,7 +15,8 @@
           'class': 'content-l_col-1-2 m-form-field__lg-target a-yes-route-mode',
           'name':  "yes-transportation-option",
           'label': label[0],
-          'value': label[0]
+          'value': label[0],
+          'disabled': true
```

Comment:
Just checking, this should be `true` and not the attribute value of `'disabled': 'disabled'`, right?

Reply:
The input atom interprets the presence of the `disabled` key with a non-falsey value as `disabled` on the element, so I believe either are fine.